content/src/denylist/DeactivatedDenylist.ts

```diff
@@ -0,0 +1,49 @@
+import { DenylistRepository } from '../repository/extensions/DenylistRepository'
+import {
+  Denylist,
+  DenylistMetadata,
+  DenylistOperationResult,
+  DenylistOperationStatus,
+  DenylistValidationType
+} from './Denylist'
+import { DenylistTarget, DenylistTargetId, DenylistTargetType } from './DenylistTarget'
+
+export class DeactivatedDenylist extends Denylist {
+  async addTarget(target: DenylistTarget, metadata: DenylistMetadata): Promise<DenylistOperationResult> {
+    return {
+      status: DenylistOperationStatus.ERROR,
+      type: DenylistValidationType.CONFIGURATION,
+      message: 'Enable target from denylist is not activated.'
```

Comment:
Could you make this a little bit clearer? Maybe something like "Denylist is not activated, so you can't add a target"